projects/packages/forms/package.json

```diff
@@ -76,6 +79,7 @@
 		"@wordpress/babel-plugin-import-jsx-pragma": "5.19.0",
 		"@wordpress/browserslist-config": "6.19.0",
 		"@wordpress/date": "5.19.0",
+		"@wordpress/scripts": "30.12.0",
```

Comment:
Also please don't bring in `@wordpress/scripts`. It brings in different versions of a lot of deps that may conflict with the ones we use, while meanwhile we generally have our own way of doing what it tries to do.

Reply:
> please don't bring in @wordpress/scripts.

@anomiex We are using the wordpress/scripts for the module support. Does the current Jetpack build script have the ability to build modules?

Reply:
Since `wp-scripts build` just calls Webpack with a predefined config, you should be able to do the same thing by setting your webpack.config.js appropriately.

At a quick glance, it looks like you may need to set [`experiments.outputModule`](https://webpack.js.org/configuration/experiments/#experimentsoutputmodule) and [`output.module`](https://webpack.js.org/configuration/output/#outputmodule). Maybe also [`output.chunkFormat = 'module'`](https://webpack.js.org/configuration/output/#outputchunkformat), [`output.environment.module`](https://webpack.js.org/configuration/output/#outputenvironment), and [`output.library.type = 'module'`](https://webpack.js.org/configuration/output/#outputlibrarytype) if setting `output.module` doesn't already change the defaults for those.